ush/python/pygfs/task/snow_analysis.py

```diff
@@ -116,9 +203,11 @@ def prepare_IMS(self) -> None:
         try:
             exe()
         except OSError:
-            raise OSError(f"Failed to execute {exe}")
-        except Exception:
-            raise WorkflowException(f"An error occured during execution of {exe}")
+            logger.exception(f"Failed to execute {exe}")
+            raise
+        except Exception err:
```

Comment:
```suggestion
        except Exception as err:
```
Add `as` before `err`, because `except Exception err` cause the crash at the early `stage_ic` jobs.